.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+Fixes # .
```

Comment:
Do we want "Fixes" as part of the template? Just wondering if community contributors who aren't familiar with GitHub know this closes the issue that is referenced once the PR is merged, especially if the PR doesn't fully fix the open issue. I don't know if it's more troublesome to reopen a issue that shouldn't be closed, or for us to go close issues if they aren't tagged to automatically be closed by a PR that does so.